sync.py

```diff
@@ -0,0 +1,166 @@
+import psycopg2
+import json
+from init import driveinit
+import concurrent.futures
+import datetime
+
+
+today = datetime.datetime.today()
+date_today = today.strftime("%Y-%m-%d")
+filesSynced = 0
+
+
+try:
+
+    connection = psycopg2.connect(database="studyportal", 
+                            user = "studyportal", 
+                            password = "studyportal", 
+                            host = "localhost", 
+                            port = "5432")
```

Comment:
did you try connecting this with the postgres running in the docker container?

Reply:
yes, got connected with postgres in docker only